main-command/src/main/scala/sbt/Watched.scala

```diff
@@ -148,6 +276,112 @@ object Watched {
     }
   }
 
+  /**
+   * Provides a number of configuration options for continuous execution.
+   */
+  trait Config {
```

Comment:
If this is meant to be used by the build user, I think it's better to move it out of `object Watched`, and name it `WatchConfiguration` or something.

Reply:
I guess for me it's a bit of a style thing. I find it intuitive to scope the kind of Configuration by putting it inside of the companion object rather than using semantic scoping with a name prefix. But I don't feel strongly enough to argue against making this change.